lib/CL/pocl_llvm_api.cc

```diff
@@ -274,6 +274,11 @@ int pocl_llvm_build_program(cl_program program,
     }
   }
 
+#ifdef HSA_RUNTIME_IS_ROCM
```

Comment:
This again assumes ROCM-based hsa device is the only device in the pocl platform.